awx_collection/tests/integration/targets/tower_credential/tasks/main.yml

```diff
@@ -369,7 +392,7 @@
       - result is failed
       - "'Unable to create credential {{ vault_cred_name2 }}' in result.msg"
       - "'Additional properties are not allowed' in result.msg"
-      - "'\\'vault_password\\' was unexpected' in result.msg"
+      - "'\\'vault_********\\' was unexpected' in result.msg"
```

Comment:
This change needs to be reverted (otherwise the test playbook fails).

Reply:
I changed this to work locally and thought assumed it would do the same on the regular test, Reverted the change.